Sources/PointFree/TODO.swift

```diff
@@ -313,3 +300,122 @@ extension PartialIso where A == B.RawValue, B: RawRepresentable {
 public func mapExcept<E, F, A, B>(_ f: @escaping (Either<E, A>) -> Either<F, B>) -> (EitherIO<E, A>) -> EitherIO<F, B> {
   return { $0.mapExcept(f) }
 }
+
+public protocol TaggedType {
```

Comment:
only way I could get the `.tagged` partial iso is to put a protocol in front of it.